i3pystatus/mpd.py

```diff
@@ -58,8 +58,8 @@ class MPD(IntervalModule):
     truncate_fields = ("title", "album", "artist")
     hide_inactive = False
     on_leftclick = "switch_playpause"
-    on_rightclick = "next_song"
```

Comment:
I think these can be configured in your config, maybe best to leave these as they are for backwards compatibility?